src/pages/fallback/NotFoundProject.tsx

```diff
@@ -26,7 +28,7 @@ export const NotFoundProject = () => {
           {t('This project does not exist')}
         </Text>
         <VStack
-          width={600}
+          width={isMobile ? '100%' : '600px'}
```

Comment:
we're emphasising the user of : 

```suggestion
          width={{base:  '100%', lg : '600px'}}
```

Over the isMobile, as it is a step slower compared to the inline method of handling different sizes. Note: `lg` is the breakpoint where `isMobile` turns false.